### PR TITLE
Random Password Generator Added

### PR DESCRIPTION
**Pull request by joshi-kaushal**
## Related Issue
  -Created a password generator that generates a random password containing when length and criteria is selected.  

### Checklist:
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
image

### CODE REVIEW

**Review by Rizwan-S — CHANGES_REQUESTED** [review]
Please remove 'linechart.html'. If you wish to add linechat, please make a separate pr. Multiple issues solved in a single pr will cause problem in allocating points. As a result, you'll get points only for a single issue.  Also, it is a bad practice in open source to make a pr for an issue assigned to others. Please take care of this from next time :)

**Comment by joshi-kaushal** [comment]
Extremely sorry, I'll make the changes ASAP

**Comment by joshi-kaushal** [comment]
@Rizwan-S please have a look

**Comment by Rizwan-S** [comment]
I think you have removed the wrong file

**Review by joshi-kaushal — COMMENTED** [review]
I have deleted a `linechart.html` file so please have a look

File: script.js [hunk 1]
```diff
@@ -2041,17 +2041,10 @@ let projectData = [
...
-    projectName: "Bar Chart",
-    projectImage: "assets/images/barchart.png",
-    projectUrl: "public/barchart.html",
-  },
-  {
-    projectName: "Scatter Chart App",
-    projectImage: "assets/images/scatter_chart.png",
-    projectUrl: "public/scatter_chart.html",
-
-  },
```
Comment (Rizwan-S) [inline script.js:2054]: Please don't remove this code.

**Review by Rizwan-S — CHANGES_REQUESTED** [review]

**Comment by joshi-kaushal** [comment]
@Rizwan-S please have a look, `git pull` might have caused addition of some updated information

**Review by Rizwan-S — APPROVED** [review]
Excellent!!!

**Pull request merged**